ParkingZoneApp/Areas/Admin/Controllers/ParkingZonesController.cs

```diff
@@ -19,8 +19,8 @@ public ParkingZonesController(IParkingZoneService parkingZoneService)
         // GET: Admin/ParkingZones
         public IActionResult Index()
         {
-            var entity = _parkingZoneService.GetAll();
-            return View(entity);
+            var parkingZones = _parkingZoneService.GetAll();  
+            return View(new ListItemVM().MapToModel(parkingZones));
```

Comment:
do not do like that create a variable and assign to it then return

Reply:
ok